apps/storybook/src/TiledHeatmap.stories.tsx

```diff
@@ -0,0 +1,208 @@
+import {
+  getLevelsSize,
```

Comment:
As per our earlier discussion, I feel like this should be called `getLevelSizes`, since it returns `Size[]`. Same with the `levelsSize` instance variables/props/... throughout.

Reply:
Now that I think of it, I find the term "level" ambiguous. I assume it comes from "level of detail"? Here, I feel like the term "layer" would make more sense, so `getLayerSizes` and `layerSizes`.